### DISCUSSION TITLE
Could BirdNET-Pi support Raspberry Pi 5?

### DISCUSSION THREAD

**Discussion by flexmox** [mcguirepr89/BirdNET-Pi#1081] [Q&A] [answered]
I recently obtained a Pi 5 and have been trying to install BirdNET-Pi unsuccessfully.  After running the installation script (curl -s https://raw.githubusercontent.com/mcguirepr89/BirdNET-Pi/main/newinstaller.sh | bash) the Pi succeds at installing then reboots (as expected) but BirdNET-Pi isn't started and navigating to the Pi shows the default Apache congrats page after apache gets installed.  I've tried running different scripts in the scripts folder myself to see if I can get it to work but I haven't been able to get it working.  Anyone else have experience with the Pi 5?

**Comment by Svardsten53** [comment, marked as answer]
The issue with RPi5 is discussed here: [https://github.com/mcguirepr89/BirdNET-Pi/issues/1068](url)

**Reply by Vollpflock** [reply]
Could someone provide a link that works? And has the problem actually been solved?

**Reply by Svardsten53** [reply]
I don't know why the link content is changed when you click on it, but if you try to type it manually you will find the issue.

**Reply by pddpauw** [reply]
I've been digging in this as well, and found one error in the Caddyfile which solves already one thing.  Still, getting an "access denied" error.  Might have to do with permissions, but couldn't find it yet.  Full description in issue 1068.  Hope someone can help!
Seems like the Analyzer is running in the background though.

**Reply by cmurtaugh** [reply]
Here's a link to issue #1068.
